sdk/python/core/docsgen/faq.rst

```diff
@@ -0,0 +1,62 @@
+FAQ
+====
+
+
+I have a question. Please help me!
+----------------------------------
+
+Please join the `YDK community <https://communities.cisco.com/community/developer/ydk>`_ to get answers to any YDK-related questions you may have.
+
+How do I report a bug I found?
+------------------------------
+
+Please use the `YDK-Gen GitHub issue tracker <https://github.com/CiscoDevNet/ydk-gen/issues>`_ to report any bugs you discovered. Please use the `YDK community <https://communities.cisco.com/community/developer/ydk>`_ for any questions.
+
+I am interested in contributing. How do I do this?
+--------------------------------------------------
+
+YDK is an open-source project and we welcome contributions. Please refer to this `brief guide <https://github.com/CiscoDevNet/ydk-gen/blob/master/CONTRIBUTIONS.md>`_, make the changes on your fork of `ydk-gen <https://github.com/CiscoDevNet/ydk-gen>`_ and submit a pull request `here <https://github.com/CiscoDevNet/ydk-gen/pulls>`_.
+
+Can you help me change something in a model bundle API?
+-------------------------------------------------------
+
+The YDK model bundle APIs are autogenerated from `predefined sets of yang models <https://github.com/CiscoDevNet/ydk-gen/tree/9c36cd1ab5c85c4017a784dc7175c75056e7b3fd/profiles/bundles>`_. Please contact the owners of the respective yang models to request any changes.
+
+What version of YDK core can be used with YDK bundles?
+------------------------------------------------------
+
+Currently, we do not have a central location where the compatibility information between the `YDK core <https://github.com/CiscoDevNet/ydk-py/tree/master/core>`_ and the bundle APIs (for example, `openconfig <https://github.com/CiscoDevNet/ydk-py/tree/master/openconfig>`_) is maintained. In principle, going forward, we plan to maintain backward compatibility between the releases of the core and bundles.
+
+The `release notes <https://github.com/CiscoDevNet/ydk-py/releases>`_ contains some information about which bundles were changed in which release of YDK.
+
+Also, the pypi page provides some information about this under the “Requires Distributions” section. For example, see:
+
+ - https://pypi.python.org/pypi/ydk-models-cisco-ios-xr/6.3.1
+ - https://pypi.python.org/pypi/ydk-models-cisco-ios-xr/6.2.2
+
+Also, we use the `ydk-gen <https://github.com/CiscoDevNet/ydk-gen>`_ tool to generate the bundles. This tool is available for anyone to use in order to generate the bundle version in combination with ydk core version of their choice. For example, the below steps will generate & install the ``cisco-ios-xr 6.3.1`` bundle compatible with ``ydk core 0.6.2`` (assuming you have already installed the `system dependencies <https://github.com/CiscoDevNet/ydk-py#system-requirements>`_):
+
+1) Install libydk (taking CentOS as example)
```

Comment:
FAQ looks great. This section could be improved if you linked to the README containing instructions to install ydk on all the systems other than CentOS or similar documentation.

https://github.com/CiscoDevNet/ydk-gen/tree/master/sdk/cpp#quick-install